postgres/init.sls

```diff
@@ -59,6 +59,9 @@ postgresql-conf:
         {{ postgres.postgresconf|indent(8) }}
     - show_changes: True
     - append_if_not_found: True
+    {% if not postgres.postgresconf_backup|default(True) -%}
```

Comment:
Just out of curiosity, was the `default()` necessary here? Did you need it to fix something or did you just add it out of habit/whatever?